test/CXX/symex/libc++/cout.cpp

```diff
@@ -0,0 +1,9 @@
+// RUN: sh %S/compile_with_libcxx.sh "%llvmgxx" "%s" "%S" "%t" "%klee" "%libcxx_include"
```

Comment:
These tests should have something like `//REQUIRE: libcxx` otherwise they fail without libcxx.

I would also prefer to have this `compile_with_libcxx` script inlined. It looks like just two `//RUN:` lines so shouldn't be too bad.